heidrun/lc_json.rb

```diff
@@ -0,0 +1,119 @@
+build_uri = lambda do |node|
+  uri = RDF::URI(node.value)
+  uri.scheme = 'http' unless uri.scheme
```

Comment:
Is there a reason why we should prefer HTTP over HTTPS in selecting the URI scheme?